src/main/resources/META-INF/resources/primefaces/fileupload/2-fileupload.js

```diff
@@ -225,13 +247,55 @@ PrimeFaces.widget.FileUpload = PrimeFaces.widget.BaseWidget.extend({
                 }
             },
             fail: function(e, data) {
+                if ($this.cfg.resumeContextPath) {
+                    $.ajax({
+                        url: $this.cfg.resumeContextPath + '?' + $.param({'X-File-Id' : $this.createXFileId(data.files[0])}),
+                        dataType: 'json',
+                        type: 'DELETE'
+                    });
+                }
```

Comment:
Not sure whether this should be at the start of fail-function. Think this would delete already uploaded chunks before we have the chance to resume.

Reply:
Not sure what you mean? fail function is called when user abort upload

Reply:
Fail is also called when the network-connection breaks. (eg when i simulate this via browser-devtools) 
We may move this block into the `if (data.errorThrown === 'abort')` { a few lines down.